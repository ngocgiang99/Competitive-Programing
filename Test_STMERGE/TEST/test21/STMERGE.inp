6
7 6
714913845 797549643 93767569 611929326 456413231 348350809 
116562493 24592462 849437872 830694033 307094460 306868075 
112698717 880181545 297063378 433277339 569008157 708308630 
624023172 83290227 842559763 675077773 62773271 23205831 
918343844 521675523 778316561 493833644 564296055 580183597 
570012213 718329975 805228143 283537023 232817802 461642942 
48722097 169515696 967799636 356301141 45258147 439116211 
1 2
655353066 51142847 
6 7
75334251 263670087 37472994 921799440 855421002 949514497 201488690 
646663739 142148845 741822946 824742928 333881759 503059704 15545887 
266482754 237514130 868028241 600860204 796736716 292805288 199233309 
829482688 856982259 490196406 250073024 628688591 215162940 718830034 
743831776 800913293 18332268 951447120 919592546 951680624 965455240 
598707398 137599082 775977773 707862486 934120200 259967326 609642938 
5 3
695778786 6724681 517850712 
387353477 897891548 969969490 
389618263 317526698 404015985 
165570953 208712475 683105648 
745961222 118336995 675053609 
5 3
693305236 154911046 742148208 
711102466 217362963 779894392 
146334613 427914611 743594259 
368069122 623259182 904869880 
166028426 858054251 557472848 
4 3
495912055 691903569 676097809 
547926001 895097131 764038854 
198680495 822898955 735553996 
862688386 791250292 429250800 
