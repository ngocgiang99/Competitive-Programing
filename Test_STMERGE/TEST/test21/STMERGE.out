48722097
51142847
90880138
517850712
166028426
627931295
